aio/content/guide/upgrade-lazy.md

```diff
@@ -8,43 +8,45 @@
+
 
 
 ## Preparation
 
-Before we start discussing how you can use `downgradeModule()` to create hybrid applications, there
-are things that you can do to ease the upgrade process even before you begin upgrading. Although not
-strictly necessary, preparation goes a long way! The steps are the same regardless how you upgrade,
-so go ahead and read the [Preparation](guide/upgrade#preparation) section of the main upgrade guide.
+Before discussing how you can use `downgradeModule()` to create hybrid apps, there
+are things that you can do to ease the upgrade process even before you begin upgrading. Because the steps are the same regardless of how you upgrade,
+refer to the [Preparation](guide/upgrade#preparation) section of [Upgrading from AngularJS](guide/upgrade).
 
 
-## Upgrading with ngUpgrade
+## Upgrading with `ngUpgrade`
 
-With the ngUpgrade library in Angular you can upgrade an existing AngularJS application
-incrementally, by building a hybrid application where you can run both frameworks side-by-side. In
-these hybrid applications you can mix and match AngularJS and Angular components and services and
-have them interoperate seamlessly. That means you don't have to do the upgrade work all at once,
-since there is a natural coexistence between the two frameworks during the transition period.
+With the `ngUpgrade` library in Angular you can upgrade an existing AngularJS app
+incrementally by building a hybrid app where you can run both frameworks side-by-side. In
+these hybrid apps you can mix and match AngularJS and Angular components and services and
+have them interoperate seamlessly. That means you don't have to do the upgrade work all at once as
+there is a natural coexistence between the two frameworks during the transition period.
 
 
-### How ngUpgrade Works
+### How `ngUpgrade` Works
 
 Regardless of whether you choose `downgradeModule()` or `UpgradeModule`, the basic principles of
-upgrading, the mental model behind hybrid applications and how you use the {@link upgrade/static
-upgrade/static} utilities remain the same. You can read about all that in the
-[How ngUpgrade Works](guide/upgrade#how-ngupgrade-works) section of the main upgrade guide.
+upgrading, the mental model behind hybrid apps, and how you use the {@link upgrade/static
+upgrade/static} utilities remain the same. For more information, see the
+[How `ngUpgrade` Works](guide/upgrade#how-ngupgrade-works) section of [Upgrading from AngularJS](guide/upgrade).
 
 <div class="alert is-helpful">
 
-  The [Change Detection](guide/upgrade#change-detection) sub-section only applies to applications
-  that use `UpgradeModule`. Change detection is handled differently with `downgradeModule()`.<br />
-  We still recommend reading the sub-section in order to better understand the differences and their
-  implications.
+  The [Change Detection](guide/upgrade#change-detection) section of 
+  the [Upgrade](guide/upgrade) guide only applies to apps 
```

Comment:
Not `of [Upgrading from AngularJS](guide/upgrade)`?

Reply:
You are correct, sir. Fixed.